src/repositories_rust.tf

```diff
@@ -44,12 +44,18 @@ locals {
       ".gitignore" = "templates/rust-all/.gitignore.tftpl"
     })
 
-    files = {
-      ".editorconfig" = {
-        content = file("templates/rust-all/.editorconfig")
+    files = merge(local.files, {
+      ".make/docker.mk" = {
+        content = file("templates/all/.make/docker.mk")
       },
-      ".cspell.config.yaml" = {
-        content = file("templates/rust-all/.cspell.config.yaml")
+      ".make/python.mk" = {
+        content = file("templates/all/.make/python.mk")
+      },
+      ".make/rust.mk" = {
+        content = file("templates/all/.make/rust.mk")
+      },
+      ".make/toml.mk" = {
+        content = file("templates/all/.make/toml.mk")
       },
       ".taplo.toml" = {
```

Comment:
cspell and editorconfig .mk?

Reply:
they are part of the default file list: https://github.com/Arrow-air/tf-github/pull/40/files#diff-1956abf05e8db0150fc12428093f137d1e9cca0a1d2d780a890c054274fb1e30R22 which will be merged into this